wdfn-server/config.py

```diff
@@ -21,6 +21,7 @@
 
 DEBUG = False
 
+SERVER_SERVICE_ROOT = 'https://waterservices.usgs.gov'
```

Comment:
I'm at a loss for a more descriptive name... perhaps a comment about how this is used by the Python components and how `SERVICE_ROOT` is used by the client side components? I could just see myself looking at this a several months and being confused.

Reply:
👍